packages/docs-app/src/examples/core-examples/buttonGroupPopoverExample.tsx

```diff
@@ -73,13 +73,13 @@ export class ButtonGroupPopoverExample extends BaseExample<IButtonGroupPopoverEx
         ];
     }
 
-    private renderButton(text: string, iconName: IconName) {
+    private renderButton(text: string, icon: IconName) {
```

Comment:
ok so actually this one was fine as `iconName` cuz it's a parameter name that matches its type. but whatever.